### PR TITLE
Merge flask-graphql

### PR DESCRIPTION
**Pull request by KingDarBoja**
References #34, this PR merges `flask-graphql` library into `graphql-server-core`.

To install the integration with flask for example:

`pip install graphql-server-core[flask]`

Then you can use it by:

```python
from grapqhl_server.flask import GraphQLView
```

### CODE REVIEW

File: graphql_server/__init__.py [hunk 1]
```diff
@@ -230,11 +231,11 @@ def get_response(
     as a parameter.
     """
 
-    if not params.query:
-        raise HttpQueryError(400, "Must provide query string.")
-
     # noinspection PyBroadException
     try:
+        if not params.query:
+            raise HttpQueryError(400, "Must provide query string.")
```
Comment (jkimbo) [inline graphql_server/__init__.py:237]: Why move this inside the try catch block? Won't that mean that the response will just return None?

File: graphql_server/flask/blueprint.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+from .graphqlview import GraphQLView
+
+
+class GraphQL(object):
+    def __init__(self, app, schema, **options):
+        self.app = app
+        warnings.warn(
+            "GraphQL Blueprint is now deprecated, please use GraphQLView directly"
+        )
```
Comment (jkimbo) [inline graphql_server/flask/blueprint.py:13]: If this is deprecated in the current flask-graphql package then this seems like a good time to remove it completely.

File: tests/flask/test_graphqlview.py [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
```
Comment (jkimbo) [inline tests/flask/test_graphqlview.py:16]: These are sims for python 2 compatibility that we can now drop.

File: graphql_server/__init__.py (hunk 1 above)
Comment (KingDarBoja) [reply graphql_server/__init__.py:237]: If you check the original code, before merging #36, the `execute_graphql_request` function was wrapped inside [execute call](https://github.com/graphql-python/graphql-server-core/pull/36/files#diff-0b0d844d3af70da5de18ae457e73d2c2L329), which also was inside the try-catch block on `get_response` function. 

The previous behaviour returned **None** if `catch_exc` was true and without putting it inside the try-catch, you will always get this error if any of the response in the list had no params, which isn't the desired behaviour.

File: graphql_server/flask/blueprint.py (hunk 2 above)
Comment (KingDarBoja) [reply graphql_server/flask/blueprint.py:13]: YEET! Consider it done!

File: graphql_server/__init__.py (hunk 1 above)
Comment (jkimbo) [reply graphql_server/__init__.py:237]: Oh sorry I misread the line `except catch_exc`. My bad

**Pull request merged**